ports/backward-cpp/portfile.cmake

```
@@ -1,21 +1,17 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO bombela/backward-cpp
-    REF "v${VERSION}"
-    SHA512 db0256a54819952ff1d92e05d6ab81fe979d4826ebb6651b6b08c30e7a0091879dfeff33d81f9599462152ce68e61e2c8c42bf039129bc6b28d1e68b1eab039b
+    REF 51f0700452cf71c57d43c2d028277b24cde32502
+    SHA512 8831be0a1c0a2f2d2625c5e2065202445520e0f7591cdbf998c60f2c892880d1527f304f9361b057c831f84621be636e12fea018297c9a17d858ecc6c36c9ffb
     HEAD_REF master
-    PATCHES
-        include-dir.diff
 )
 
-set(VCPKG_BUILD_TYPE release) # header-only
-
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
 )
 vcpkg_cmake_install()
-vcpkg_cmake_config_fixup(PACKAGE_NAME backward CONFIG_PATH lib/backward)
+vcpkg_cmake_config_fixup(PACKAGE_NAME backward-cpp CONFIG_PATH lib/cmake/backward)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/backward)
```
 - The actual package name is `Backward`, 
 - the default `PACKAGE_NAME` is the port name `backward-cpp`,
 - CMake looks for the config in `share/<lower-case-package>*`.

So the default is fine for us.

---

The name "backward" was provided by the original repository author in the cmake file, but his repository name is called "backward-cpp", and I believe that the port name in vcpkg should be consistent with the repository name.

---

Focus on the suggestion. I didn't request changes to the port name or CMake package name. 
(`PACKAGE_NAME` is a misnomer. It only determines the directory name in `share`.)